server/webapp/WEB-INF/rails.new/webpack/views/dashboard/pipeline_analytics_widget.js.msx

```diff
@@ -24,11 +24,16 @@ const PluginEndpointRequestHandler = require('rails-shared/plugin-endpoint-reque
 const Frame                        = require('models/shared/frame');
 
 
+const models = {};
+PluginEndpointRequestHandler.defineLinkHandler();
+PluginEndpointRequestHandler.defineFetchAnalyticsHandler(models);
```

Comment:
For the pipeline analytics, I assume we don't require linkHandler, but we will need FetchAnalyticsHandler for the charts with configurable time. (Eg. data for a week, month, etc)?